core/src/components/setup/RecommendedApps.vue

```diff
@@ -215,7 +228,9 @@ p {
 	img {
 		height: 50px;
 		width: 50px;
-		filter: invert(1);
+		@media (prefers-color-scheme: dark) {
+			filter: invert(1);
+		}
```

Comment:
```suggestion
		filter: var(--background-invert-if-dark);
```

To respect the user preference in case they overwrite the browser prefers setting?
Or does it not work on the install/update page?